examples/cfd/08_shallow_water_equation.ipynb

```diff
@@ -0,0 +1,4210 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        eta : numpy.ndarray
</pre><p>Should these all be Devito <code>Function</code>s? The code implies they are  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Should typically be </p><p><code>ndarray or Function</code></p>

Reply:
<p>Oh sorry, I forgot to adapt it when I rewrote the function, they all are devito Functions yes.</p>